broker-daemon/broker-rpc/info-service/get-trades.js

```diff
@@ -0,0 +1,41 @@
+const { MarketEvent } = require('../../models')
+
+/**
+ * Default limit for number of records returned per call
+ * @type {Integer}
+ * @constant
+ */
+const DEFAULT_LIMIT = 50
```

Comment:
[nit] The number we pass in for the limit is `int64`. It's a good practice to be consistent and have this as `Big(50)`.